app/views/tube_creation/pooled_tubes_by_sample.html.erb

```diff
@@ -21,12 +15,12 @@
           <tr><td>A2</td><td>AB00000009</td></tr>
           <tr><td>B2</td><td>AB00000010</td></tr>
           <tr><td>C2</td><td>AB00000011</td></tr>
-          <tr><td>D2</td><td>NoScan</td></tr>
-          <tr><td>E2</td><td>NoScan</td></tr>
+          <tr><td>D2</td><td>NO READ</td></tr>
+          <tr><td>E2</td><td>AB00000013</td></tr>
           <tr><td>etc...</td></tr>
         </tbody>
       </table>
-      <p>NB. Columns headers and all 96 positions in the rack must be represented in the file. If a tube is not present in a position, the system expects to see a NoScan value in the barcode column.</p>
+      <p>NB. The system will also understand the value NO READ instead of a barcode, if the scan was not successful in that position, and ignore that tube when deciding where to transfer material.</p>
```

Comment:
Sorry I don't fully understand this sentence, particularly the `, if the scan was not successful in that position,` bit. Might just be me

Reply:
OK thanks, I will try to re-word it

Reply:
Updated - any better?